pkg/sqlutil/log.go

```diff
@@ -0,0 +1,227 @@
+package sqlutil
```

Comment:
This PR made me think of `core/services/pg/sqlx.go` - did you imagine we would consolidate the `Queryer` interface there into the one in this package?

Reply:
Yeah, this is meant to replace it.